src/vs/workbench/contrib/localization/electron-sandbox/localization.contribution.ts

```diff
@@ -210,9 +211,7 @@ export class LocalizationWorkbenchContribution extends Disposable implements IWo
 	private async isLocaleInstalled(locale: string): Promise<boolean> {
 		const installed = await this.extensionManagementService.getInstalled();
 		return installed.some(i => !!(i.manifest
-			&& i.manifest.contributes
-			&& i.manifest.contributes.localizations
-			&& i.manifest.contributes.localizations.length
+			&& i.manifest.contributes?.localizations?.length
```

Comment:
might as well combine the `i.manifest` and `i.manifest.contributes?.localizations?.length`